### ISSUE TITLE
[suggestion] Disable pass-through and limbo config

### ISSUE THREAD

**Issue by xpiREC** [kyngs/LibrePremium#12]
It would be very nice to see an option to disable redirection of players after login and limbo, the reason for this is that I have several sub servers that are connected via proxy and the selection is done by the version of the client that connects to the server (That is:  VersionConnector) 
This option is helpful for small networks which don't want to run additional servers just to move players and chewing up extra resources for unnecessary things.

**Comment by kyngs** [comment]
How does the VersionConnector find out when to connect the player to the lobby or limbo?

**Comment by xpiREC** [comment]
No, VersionConnector only connects players by detecting their version when connecting, and since LibrePremium does additional redirection, this is not possible, hence the question about adding the ability to disable this functionality.

**Comment by kyngs** [comment]
I guess I might add it, but I would suggest you to use the API for this.

**Comment by xpiREC** [comment]
Unfortunately I'm not that good at Java programming, no thanks to me though.

**Comment by xpiREC** [comment]
Unfortunately I'm not that good at Java programming.
Therefore, I have created are suggestions that may also be helpful to others in the future.

**Comment by HarvelsX** [comment]
> Be the very last one to make the determination and if the user needs to be rerouted to an authorization server, remember where he should have connected before and reroute to where you need to go with status tracking.

The plugin can still save its implementation of player destination for connections that do not have an end server.
Users can remove all destinations in the configuration of the proxy server, if possible, otherwise put a stub, which the plugin will perceive as no server. And in this case the server destination will be delegated by the plugin.
